third_party/move/move-compiler/tests/move_check/parser/aptos_stdlib_attributes.move

```diff
@@ -0,0 +1,8 @@
+module aptos_std::M {
+    #[a, a(x = 0)]
+    fun foo() {}
+
+    #[testonly]
```

Comment:
This test case is really hard to understand, I think you are testing the warning suppression here. Perhaps name the module `aptos_std::module_with_suppressed_warnings`. BTW newer Move naming conventions have modules in small letters.